logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -551,6 +555,42 @@ internal class MLSConversationDataSource(
             }
         }
 
+    override suspend fun getUserIdentity(userId: UserId) =
+        wrapStorageRequest { conversationDAO.getMLSGroupIdByUserId(userId.toDao()) }.flatMap { mlsGroupId ->
+            mlsClientProvider.getMLSClient().flatMap { mlsClient ->
+                wrapMLSRequest {
+                    mlsClient.getUserIdentities(
+                        mlsGroupId,
+                        listOf(userId.toCrypto())
+                    )[userId.value]!!
+                }
+            }
+        }
+
+    override suspend fun getMembersIdentities(conversationId: ConversationId): Either<CoreFailure, Map<UserId, List<WireIdentity>>> =
```

Comment:
question: when we fetch the users, are they in the same order we show in the UI for the conversation's member?

Reply:
I'm not user about order in List, that why used `Map<UserId, List<WireIdentity>>` here :)